components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -577,6 +584,16 @@ - (void)setTitlePositionAdjustment:(UIOffset)titlePositionAdjustment {
   }
 }
 
+- (void)setTitleNumberOfLines:(NSInteger)titleNumberOfLines {
+  _titleNumberOfLines = titleNumberOfLines;
+  self.label.numberOfLines = self.titleBelowIcon ? titleNumberOfLines : 1;
+}
+
+- (void)setTitleBelowIcon:(BOOL)titleBelowIcon {
+  _titleBelowIcon = titleBelowIcon;
+  self.label.numberOfLines = self.titleBelowIcon ? self.titleNumberOfLines : 1;
```

Comment:
Please extract this repeated line (589 being the other) to a private method.

Reply:
Done